usb-gadget.md

```diff
@@ -0,0 +1,19 @@
+# USB HID gadget
+
+Linux devices with host and either USB OTG or device port can be used as USB to USB converter boards, with keyboard connected to USB host port and PC to USB OTG or device port.
+
+In that kind of setup Linux USB HID gadget driver can be used to emulate HID device and `keyd` can be configured to translate evdev input events to HID reports.
+
+
+# Installation
+
+    sudo apt-get install libudev-dev # Debian specific, install the corresponding package on your distribution
+
+    git clone https://github.com/rvaiya/keyd
+    cd keyd
+    make vkbd-usb-gadget && sudo make install && sudo make install-usb-gadget
```

Comment:
I would call `install` from `install-usb-gadget` and roll this into one target

Reply:
Updated. Still feels like there is shorter way to do it, so fill free to correct me again.